jstreegrid.js

```diff
@@ -1032,22 +1123,28 @@
 				a.addClass(c);
 				//renderAWidth(a,_this);
 				renderATitle(a,t,_this);
-				last = a;
+        last = a;
+
+        // calculate position ids once
+        gridCellPrevId = pos <= 0 ? objData.parent : findLastClosedNode(this, peers[pos - 1]);
+        gridCellNextId = pos >= peers.length - 1 ? "NULL" : peers[pos + 1];
+        gridCellChildId = objData.children && objData.children.length > 0 ? objData.children[0] : "NULL";
+
 				// find which column our tree shuld go in
 				var s = this.settings.grid;
-				for (i=0;i<cols.length;i++) {
+        for (var i = 0, len = cols.length;i<len;i++) {
 					if (treecol === i) {
 						continue;
 					}
 					col = cols[i];
-					dataCell = mw.children("div:eq("+i+")");
+          column = $(this._domManipulation.columns[i]); //Geht the detached column not mw.children("div:eq("+i+")");
 					// get the cellClass, the wideCellClass, and the columnClass
 					cl = col.cellClass || "";
 					wcl = col.wideCellClass || "";
 					ccl = col.columnClass || "";
 
-					// add a column class to the dataCell
-					dataCell.addClass(ccl);
+					// add a column class to the Column
+          column.addClass(ccl);
```

Comment:
Similar: if `column` is just the equivalent of `dataCell` but using the detached, then I understand this.